recipes/hugo/meta.yaml

```
@@ -0,0 +1,34 @@
+{% set name = "hugo" %}
+{% set version = "0.122.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gohugoio/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 7468280290ce43276e26f964b0cab234d2bf0546f80295890189054b98ffac38
+
+build:
+  number: 0
```

This is known to avoid segfaults.
```suggestion
  number: 0
  binary_relocation: false
```